src/js/video.js

```diff
@@ -199,6 +199,7 @@ class Video {
 			this.videoEl.style.display = 'block';
 			if (!this.opts.advertising) {
 				this.videoEl.load();
+				this.videoEl.play();
```

Comment:
it should work without the play, load just plays the video from the beginning if I understood correctly